UITests/TransferFunds/TransferUserFundsTest.swift

```diff
@@ -0,0 +1,548 @@
+        XCTAssertTrue(transferFunds.transferFundTitle.exists)
+        XCTAssertTrue(transferFunds.addSelectDestinationSectionLabel.exists)
+        XCTAssertEqual(transferFunds.addSelectDestinationLabel.label, "Bank Account")
+
+        if #available(iOS 11.0, *) {
+            XCTAssertEqual(transferFunds.addSelectDestinationDetailLabel.label, "United States\nEnding on 1234")
+        } else {
+            XCTAssertEqual(transferFunds.addSelectDestinationDetailLabel.label, "United States Ending on 1234")
+        }
+        // Transfer Section
+        XCTAssertTrue(transferFunds.transferSectionLabel.exists)
+        // Amount row
+        XCTAssertEqual(transferFunds.transferAmountLabel.label, "Amount")
+        XCTAssertEqual(transferFunds.transferCurrency.label, "USD")
+        // Transfer all funds row
+        XCTAssertEqual(transferFunds.transferAllFundsLabel.label, "Transfer all funds")
+        XCTAssertTrue(transferFunds.transferAllFundsSwitch.exists, "Transfer all funds switch should exist")
+
+        // Notes
+        XCTAssertTrue(transferFunds.notesSectionLabel.exists)
+        XCTAssertEqual(transferFunds.notesDescriptionTextField.placeholderValue, "Description")
+
+        let availableFunds = app.tables["createTransferTableView"].staticTexts["Available for transfer: 452.14"]
+        XCTAssertTrue(availableFunds.exists)
+
+        // Next Button
+        XCTAssertTrue(transferFunds.nextLabel.exists)
+    }
+
+    /*
+     Given that Transfer methods exist
+     When user select a different transfer method with different currency
+     Then the destination currency should be updated AND Then Payee can enter the amount
+     */
+    func testTransferFunds_switchTransferMethod_Currency() {
+        mockServer.setupStub(url: "/rest/v3/users/usr-token/transfer-methods",
+                             filename: "ListMoreThanOneTransferMethod",
+                             method: HTTPMethod.get)
+
+        mockServer.setupStub(url: "/rest/v3/transfers",
+                             filename: "AvailableFundUSD",
+                             method: HTTPMethod.post)
+
+        XCTAssertTrue(transferFundMenu.exists)
+        transferFundMenu.tap()
+        waitForNonExistence(spinner)
+
+        XCTAssertTrue(transferFunds.addSelectDestinationSectionLabel.exists)
+        XCTAssertEqual(transferFunds.addSelectDestinationLabel.label, "Bank Account")
+        // Amount row
+        XCTAssertEqual(transferFunds.transferAmountLabel.label, "Amount")
+        XCTAssertEqual(transferFunds.transferCurrency.label, "USD")
+
+        transferFunds.addSelectDestinationLabel.tap()
+        // Select Destination (CAD)
+        let cadBankAccount = app.tables.element.children(matching: .cell).element(boundBy: 1)
+        cadBankAccount.tap()
+        XCUIApplication().statusBars.children(matching: .other).element.children(matching: .other).element(boundBy: 0).tap()
+
+        XCTAssertEqual(transferFunds.transferCurrency.label, "CAD")
```

Comment:
Can you add a response and assertion for the quote as well after switching destinations?